.github/workflows/pr.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           echo "\`\`\`diff" >> ${GITHUB_STEP_SUMMARY}
           diff -u0 branch.errors main.errors | grep '^-[^-]' | tee -a ${GITHUB_STEP_SUMMARY}
           echo "\`\`\`" >> ${GITHUB_STEP_SUMMARY}
-          if [[ $(diff -u0 branch.errors main.errors | grep -c '^-[^-]') != 0 ]];  then
+          if grep -q '^-[^-]' <<< $(diff -u0 branch.errors main.errors); then
```

Comment:
that's an interesting way to redirect the output, I've always used something like `grep foo <(command)` because that creates a temp file descriptor and treats it as a file

Reply:
No problem doing this, but I submitted the change before reading your previous comment, sending it this way now